app/services/app_store_update_processor.rb

```diff
@@ -18,10 +18,13 @@ def convert_params(record)
       }
     end
 
-    def update_claim(claim_id, params)
-      claim = Claim.find_by(id: claim_id)
+    def update_claim(record)
+      claim = Claim.find_by(id: record['application_id'])
 
-      claim&.update!(params)
+      return unless claim
+
+      claim.update!(convert_params(record))
+      Nsm::AssessmentSyncer.call(claim, record:)
     end
 
     def update_prior_authority_application(application_id, params, record)
```

Comment:
@ivanELEC for consistency of interfaces it feels like this method should be changed to do the same as `update_claim`. But I have already approved so leave it up to you

Reply:
good shout - just added this in